api/maven-api-core/src/main/java/org/apache/maven/api/Project.java

```diff
@@ -37,24 +37,58 @@
 @Experimental
 public interface Project {
 
+    /**
+     * Returns the project groupId.
+     */
     @Nonnull
     String getGroupId();
 
+    /**
+     * Returns the project artifactId.
+     */
     @Nonnull
     String getArtifactId();
 
+    /**
+     * Returns the project version.
+     */
     @Nonnull
     String getVersion();
 
+    /**
+     * Returns the project packaging.
+     */
     @Nonnull
     String getPackaging();
 
+    /**
+     * Returns the project artifact, that is the artifact produced by this project. If the artifact file exists, it is
+     * registered to {@link org.apache.maven.api.services.ArtifactManager}.
+     *
+     * @see org.apache.maven.api.services.ArtifactManager#getPath(Artifact)
+     */
     @Nonnull
```

Comment:
This should tell whether this is the main artifact or the POM. Must be in send with the plural method.

Reply:
This is "the" artifact that project produces (will install, deploy, whatever). IMHO, it should not tell that, and should remain agnostic to all those... IF someone wants POM artifact or BUILD artifact, then there is new method...